cmd/triggerRun/testdata/trigger.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: pipeline-binding
+spec:
+  params:
+  - name: foo
```

Comment:
The Trigger spec does not contain params. We should use a valid Trigger e.g. from the Triggers section of an existing event listener:  https://github.com/tektoncd/triggers/blob/master/examples/eventlisteners/eventlistener.yaml#L8